modules/govuk_postgresql/manifests/wal_e/backup.pp

```diff
@@ -60,7 +60,8 @@
   $archive_timeout = '300',
   $hour = 6,
   $minute = 20,
-  $db_dir = '/var/lib/postgresql/9.3/main',
+  $db_dir = $postgresql::globals::datadir,
+  $enabled = false,
```

Comment:
Oh, that's interesting. I think it got added when I rebased it, it shouldn't be there.